src/display/editor/freetext.js

```diff
@@ -648,6 +648,14 @@ class FreeTextEditor extends AnnotationEditor {
     }
   }
 
+  #serializeContent() {
+    return this.#content.replaceAll("\u00a0", " ");
+  }
+
+  static #deserializeContent(content) {
+    return content.replaceAll(" ", "\u00a0");
+  }
+
```

Comment:
This file uses one format and the unit-tests another, can we please be consistent here (preferable using the shorter format)?
```suggestion
  #serializeContent() {
    return this.#content.replaceAll("\xa0", " ");
  }

  static #deserializeContent(content) {
    return content.replaceAll(" ", "\xa0");
  }

```